src/tests/tests/firstparties.js

```diff
@@ -0,0 +1,71 @@
+(function() {
+  let tco = 'http://t.co/beach-detour/';
+  let destination = 'https://the.beach/';
+
+  document.querySelectorAllBefore = document.querySelectorAll;
```

Comment:
Should we also stub `setTimeout` to prevent the content script's timer from running continuously on the tests page?